temporal-sdk/src/test/java/io/temporal/workflow/shared/TestWorkflowWithCronScheduleImpl.java

```diff
@@ -34,6 +34,7 @@
 public class TestWorkflowWithCronScheduleImpl implements TestWorkflowWithCronSchedule {
 
   public static final Map<String, AtomicInteger> retryCount = new ConcurrentHashMap<>();
+  public static String[] lastCompletionResults = new String[10];
```

Comment:
Usually there's no more than 5 runs and we start at index 1 instead of 0. 10 to make sure there's enough space